client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -29,6 +40,15 @@ export default function ProfileDetails(): JSX.Element {
       getProfile(profileId).then((res) => {
         if (!res.error) {
           setProfile(res.success.profile);
+          getReviews(profileId).then((res) => {
```

Comment:
Let's also chain a `catch` block to handle a promise rejection.

Alternatively, it would be easier to use async/await syntax for increased readability and maintainability

Reply:
I thought that the API function was handling promise rejection with its own catch block (and setting an error), then we handle whether it returned a success or error on the front end. This is how we have been doing it on the other API helper calls, am I missing something?